server/src/main/java/org/apache/cassandra/sidecar/db/schema/SystemAuthSchema.java

```diff
@@ -66,21 +100,41 @@ protected String tableName()
                                                 "tables in system_auth keyspace");
     }
 
-    @Nullable
+    @NotNull
     public PreparedStatement selectRoleFromIdentity()
     {
+        ensureSchemaAvailable();
```

Comment:
ensureSchemaAvailable is throwing SchemaUnavailableException which a runtime exception. While the other methods whcih return the prepared statements could return null. Any reason to chose different behavior for these methods? Can it be made same?